docs/standards/getting-started/02-add-key-manager.md

```diff
@@ -0,0 +1,348 @@
+---
+sidebar_label: 'Add control with a Key Manager'
+sidebar_position: 2.2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Add control with a KeyManager
+
+In order to interact with the KeyManager, you will need to download its ABI from our npm package
+
+```javascript
+import KeyManager from '@lukso/universalprofile-smart-contracts/build/contracts/KeyManager/KeyManager.json';
+```
+
+## Setting up a Key Manager
+
+When you create a Universal Profile using the LSP Factory tools, a Key Manager is automatically deployed and ownership of your UP is automatically transfered to it.
```

Comment:
`using lsp-factory.js` -> let's write it always the same way.

`and the ownership of` maybe?

`transferred`